Grammar/python.gram

```diff
@@ -638,8 +638,13 @@ type_alias[stmt_ty]:
 # Type parameter declaration
 # --------------------------
 
-type_params[asdl_type_param_seq*]: '[' t=type_param_seq  ']' {
+type_params[asdl_type_param_seq*]: 
+    | '[' t=type_param_seq  ']' {
         CHECK_VERSION(asdl_type_param_seq *, 12, "Type parameter lists are", t) }
+    | '[' token=']' {
+        RAISE_SYNTAX_ERROR_STARTING_FROM(
+            token, 
+            "At least one type variable definition is expected")}
```

Comment:
Should this maybe be "type parameter definition" rather than "type variable definition"? TypeVarTuples and ParamSpecs can also be defined within the square brackets